desktop/linux/vm.md

```diff
@@ -0,0 +1,34 @@
+---
+description: Docker Desktop for Linux 
```

Comment:
⚠️ Looks like this page is not included in the TOC;


❓ Wondering if we should combine;

- Docker Desktop for Linux vs Docker Engine
- Why Docker Desktop for Linux runs a VM

And put them both on a more generic "Overview" or "Description" page that outlines what Docker Desktop for Linux is. That way we can add "new" information if things come, or remove sections that are no longer relevant, without having to create new pages for that

@usha-mandya @aevesdocker WDYT?